pkg/types/chains/evm/evm.go

```diff
@@ -108,3 +114,50 @@ type Head struct {
 type TransactionFee struct {
 	TransactionFee *big.Int // Cost of transaction in wei
 }
+
+type SignedReport struct {
+	RawReport     []byte
+	ReportContext []byte
+	Signatures    [][]byte
+	ID            []byte
+}
+
+// TransactionStatus is the result of the transaction sent to the chain
+type TransactionStatus int
```

Comment:
Do we need a new type for this? Other (evm) methods use a different type.

Reply:
We want a different abstraction that the one we have. TransactionResult is what's expected from a client in EVMService while the other TransactionStatus we have are: 
 - In contract_writer, which is specific to that implementation which will deprecate
 - In TXManager which has more states than what we should expose outside EVMService.